DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/EmailDelegates/Index.cshtml

```diff
@@ -67,6 +69,7 @@
   </div>
 </div>
 
+<input type="hidden" id="javascript-search-enabled" asp-for="@Model.JavascriptSearchSortFilterPaginateEnabled" />
 @section scripts {
   <script src="@Url.Content("~/js/trackingSystem/emailDelegates.js")" asp-append-version="true"></script>
 }

```

Comment:
Should this have a check for `JavascriptSearchSortFilterPaginateEnabled`?

Reply:
Just tested and it doesn't preserve the selected delegates when you interact with the filter controls, but I'm not really sure what we can do about it.
The filtering still works, and the filtered items are the only items selected if you don't mess with the filters before submitting, so it's probably good enough. I'm not sure it's likely to matter for this screen though, but we could just not disable the javascript filtering ever on this page @SteveJacksonSoft ?

Reply:
The page should function without JS, so it might be an option just to disable it all. Might need to check the original implementation ticket to see if there were any problems with the No JS version, but I reckon that's the best course of action. But keeping the JS for this page should be fine anyway.

Reply:
Is this page set up right? When I open this page with no filters at a centre with 135 active delegates it says there are only 50 matching results.

I think it's best to disable the rest of the JS if we have too many delegates on this page.

Reply:
It doesn't display all delegates, only those that are approved, non-self-registered, have an email and do not have a password.